GOVERNANCE.md

```diff
@@ -0,0 +1,146 @@
+# Debezium Governance
+
+This document defines the project governance for the [Debezium](https://github.com/debezium) project.
```

Comment:
Link to the website instead?

Reply:
All the governance documents I reviewed typically just had the project name text, so I added the link. I'm okay with adding it to the website. :+1: